core/src/test/java/io/grpc/internal/ManagedChannelImplBuilderTest.java

```diff
@@ -367,7 +366,7 @@ public void transportDoesNotSupportAddressTypes() {
     when(mockClientTransportFactoryBuilder.buildClientTransportFactory())
         .thenReturn(mockClientTransportFactory);
     when(mockClientTransportFactory.getSupportedSocketAddressTypes())
-        .thenReturn(Collections.singleton(InProcessSocketAddress.class));
+        .thenReturn(Collections.singleton(SpecialSocketAddress.class));
```

Comment:
You might want to call it `NoOpSocketAddress` since "special" implies that it does something better than normal rather than just nothing (yes, sometimes nothing is better than what's normally done, but ...).

Reply:
It isn't noop either, because socketaddress never does anything itself. "ArbitrarySocketAddress"? "RandomSocketAddress"? Ehhh... "CustomSocketAddress'? I think I like that one; it is similar to special but I think with different enough connotation.

Reply:
CustomSocketAddress sounds fine.